tracing/README.md

```diff
@@ -0,0 +1,37 @@
+# Tracing package
+
+`package tracing` provides an API wrapper around the tracing library
+[jaeger](https://github.com/uber/jaeger-client-go) that allows you to
```

Comment:
- jaeger --> Jaeger
- Which thing allows this instrumentation? Is it package tracing, the API wrapper, or Jaeger specifically? The sentence should say so more clearly.